apps/src/levelbuilder/skills/SkillEvaluationSettings.tsx

```diff
@@ -1,24 +1,34 @@
+import Link from '@code-dot-org/component-library/link';
 import React from 'react';
 import * as Table from 'reactabular-table';
 
+import {studio} from '@cdo/apps/lib/util/urlHelpers';
 import {tableLayoutStyles as style} from '@cdo/apps/templates/tables/tableConstants';
 
+import AccuracyCheck from './AccuracyCheck';
 import {columns} from './SkillsByConceptTable';
 import {Skill} from './types';
+import ViewSystemPrompt from './ViewSystemPrompt';
 
 interface Props {
   skills: Skill[];
+  levelId: number;
+  systemPrompt: string;
 }
 
-const SkillEvaluationSettings: React.FC<Props> = ({skills}) => {
+const SkillEvaluationSettings: React.FC<Props> = ({
+  skills,
+  levelId,
```

Comment:
In theory (not asking for a change, just playing a thought experiment in my head), could we have level builders pass in level names instead of ids?  This might reduce the issues with different level_ids depending on environment that I anticipate might happen...

I believe names are unique too, so it seems like it could work...

Reply:
yep, any information that could be used to determine the id will work! 

As long as levelbuilders do all of this work on levelbuilder using levelbuider level and skill ids it should be ok. I think the confusing part is in a hypothetical future if they want to pull student work since that can only be done on production.